conf/application.conf

```diff
@@ -85,7 +85,7 @@ microservice {
     secure-messaging {
       protocol = http
       host = localhost
-      port = 6790
+      port = 6790 //9055 when connecting to local running secure-messaging-frontend service
```

Comment:
Does this comment need to be a #?

Reply:
Yeah, not sure if `#` is valid mid line. I've removed the comment to be on the safe side as it was only a 'nice to have' :+1: